### PR TITLE
Add latest openshift release v3.2

### PR DESCRIPTION
**Pull request by praveenkumar**
I created this PR to test OSE-3.2 which is latest release.

### CODE REVIEW

File: build_tools/kickstarts/rhel-7-cdk-vagrant.ks [hunk 1]
```diff
@@ -55,12 +55,9 @@ openshift2nulecule
...
 # Originally taken entirely from original upstream Atomic project kickstart
-
-# TODO: Remove this as soon as BZ1281805 is resolved in the official 7.2 package set
-# It is still not resolved
-rpm -e docker-selinux
-rpm -Uvh http://download.eng.pnq.redhat.com/brewroot/packages/docker/1.9.1/25.el7/x86_64/docker-selinux-1.9.1-25.el7.x86_64.rpm
-
+# Workaround for #1336857
+chcon -t docker_exec_t /usr/bin/docker-current
+chcon -t docker_exec_t /usr/bin/docker-storage-setup
```
Comment (LalatenduMohanty) [inline build_tools/kickstarts/rhel-7-cdk-vagrant.ks]: We should use restorecon here. `restorecon -v /usr/bin/docker-storage-setup`

Comment (LalatenduMohanty) [inline build_tools/kickstarts/rhel-7-cdk-vagrant.ks]: On cdk 2.0 I do not see docker_exec_t on docker-storage-setup. 

```
$ ls -Z docker-storage-setup 
-rwxr-xr-x. root root system_u:object_r:bin_t:s0       docker-storage-setup
```

Comment (LalatenduMohanty) [inline build_tools/kickstarts/rhel-7-cdk-vagrant.ks]: @praveenkumar Looks like we are missing something here. /usr/bin should not have docker_exec_t

Comment (praveenkumar) [inline build_tools/kickstarts/rhel-7-cdk-vagrant.ks]: > We should use restorecon here. restorecon -v /usr/bin/docker-storage-setup

restorecon - restore file(s) default SELinux security contexts.
chcon - change file SELinux security context

We can use restorecon if SELinux sercurity context is set and we did some modification. Here context is not set.

> On cdk 2.0 I do not see docker_exec_t on docker-storage-setup. 

With latest release of docker-1.9.1-40.el7.x86_64, it should have that. Please check https://bugzilla.redhat.com/show_bug.cgi?id=1336857

**Comment by praveenkumar** [comment]
@LalatenduMohanty  please do review. This PR only for update OSE tag.

**Comment by bexelbie** [comment]
LGTM - did not do a test build

**Comment by hferentschik** [comment]
Do we already having a build which includes this OSE version?

**Comment by LalatenduMohanty** [comment]
LGTM. @praveenkumar please rebase and merge.

**Comment by praveenkumar** [comment]
I am closing this PR because it has been implemented as part of #402

**Pull request closed without merging**